docs/source/notebooks/dap_subset.ipynb

```diff
@@ -508,17 +508,17 @@
    "execution_count": 3,
    "metadata": {
     "execution": {
-     "iopub.execute_input": "2022-01-24T17:53:20.793856Z",
-     "iopub.status.busy": "2022-01-24T17:53:20.720158Z",
-     "iopub.status.idle": "2022-01-24T17:53:20.868994Z",
-     "shell.execute_reply": "2022-01-24T17:53:20.867291Z"
+     "iopub.execute_input": "2022-02-04T18:10:43.864083Z",
+     "iopub.status.busy": "2022-02-04T18:10:43.847732Z",
+     "iopub.status.idle": "2022-02-04T18:10:43.902766Z",
+     "shell.execute_reply": "2022-02-04T18:10:43.900886Z"
     }
    },
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'lat': 1, 'lon': 2, 'time': slice(5040, 6840, None)}"
+       "{'lat': array(1), 'lon': array(2), 'time': slice(5040, 6840, None)}"
```

Comment:
Should add a NBVAL_IGNORE_OUTPUT, else this cell will break in Jenkins that use a Jupyter env that do not have panda 1.4.0 yet.